[validate] - allow all validations to be ignored on external repos (#3309)
demisto/demisto-sdk: CHANGELOG.md

# Changelog
## Unreleased

> * Added the ability to ignore any validation in the **validate** command when running in an external (non-demisto/content) repo, using the <FILE_NAME> file.

Reviewer: please fill the file name here 🙂

> * Added the ability to ignore any validation in the **validate** command when running in an external (non-demisto/content) repo, using the <FILE_NAME> file.

Reviewer: ```suggestion
* Added the ability to ignore any validation in the **validate** command when running in an external (non-demisto/content) repo, by placing a `.private-repo-settings` file at its root.
```

* Added `tenant_timezone` handling to the **modeling-rules init** command, allowing usage with tenants in various timezones.
* Shortened the timeout when checking whether the dataset exists in **test-modeling-rule**.
* Cleaned up project dependencies.

> * Added the ability to ignore any validation in the **validate** command in case the repository is external.

Reviewer: ```suggestion
* Added the ability to ignore any validation in the **validate** command when running in an external (non-demisto/content) repo, using the <FILE_NAME> file.
```
